core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/dot-ema-shell.component.ts

```diff
@@ -229,8 +229,6 @@ export class DotEmaShellComponent implements OnInit, OnDestroy {
         this.router.navigate([], {
             queryParams,
             queryParamsHandling: 'merge'
-            // replaceUrl: true,
-            // skipLocationChange: false,
```

Comment:
Why?

Reply:
@zJaaal you did this?

Reply:
I didn't deleted that, that comment has like a month there, not sure if we want to keep it just in case. Anyways It's safe to delete it